tsconfig.base.json

```
@@ -111,6 +111,7 @@
       "@energinet-datahub/dh/shared/ui-date-time": [
         "libs/dh/shared/ui-date-time/src/index.ts"
       ],
+      "@energinet-datahub/eo-faq/routing": ["libs/eo/faq/routing/src/index.ts"],
```

issue: This import path alias doesn't follow our format. Make sure to update all existing import statements using this library.
```suggestion
      "@energinet-datahub/eo/faq/routing": ["libs/eo/faq/routing/src/index.ts"],
```